server/services/participants-helper.js

```diff
@@ -161,6 +161,20 @@ class FilteredParticipantsFinder {
         field: joinFieldName,
         direction: pagination.direction || 'asc',
       });
+
+      // To manage employer name column sorting we need to sort by employer name
+      if (sortField === 'employerName') {
```

Comment:
Nit - Might be good to reference a constant/enum here.

Reply:
yup whole function need to refactor regarding const name ....